test.nix

```diff
@@ -0,0 +1,48 @@
+{ nixpkgsBranch ? "release-21.05"
```

Comment:
What's the difference between `test.nix` and `default.nix`? Both files could do a comment in the beginning the file to explain what does the file do.